package.json

```
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "start": "node src/buildServer.js",
-    "test": "mocha 'test/apiTest.js' --exit"
+    "test": "mocha 'test/catRoutes.test.js' --exit",
```

```suggestion
    "test": "mocha 'test/catRoutes.test.js' --exit && jest"
```

I'd suggest using [bash's AND operator](https://stackoverflow.com/questions/4510640/what-is-the-purpose-of-in-a-shell-command) (&&) to run the jest tests after mocha. This has the added bonus of running them on CI and makes it harder to forget to run them.